src/main/java/com/assignment/Assignment8.java

```diff
@@ -0,0 +1,14 @@
+package com.assignment;
+
+//Printing middle number
+public class Assignment8 {
+    public static int middle(){
+        int[] arrayM = new int[500];
+        for(int i=0; i<500; i++) {
+            arrayM[i]=i+1;
+        }
+        int middleNumber = (arrayM.length)/2;
```

Comment:
If you are doing this then no need of for loop.
on line 6 you define the size of array
And in line 10 you are getting the size of array